.template/addons/svgeez/.github/wiki/template.rb

```diff
@@ -0,0 +1,14 @@
+
+# frozen_string_literal: true
+
+use_source_path __dir__
+
+copy_file 'Add-new-svg-icon.md', '.github/wiki/Add-new-svg-icon.md'
```

Comment:
1 thing, the `Github addon is an optional addon` -> `install_addon_prompt('Github Action and Wiki')` so the `.github/wiki` might not exist if the user does not select the Github Addon, I think here we need to do something like check if the directory existing or not before copy, what do you think @mosharaf13